.github/workflows/test-push.yml

```diff
@@ -0,0 +1,35 @@
+name: Remote Dispatch Action Initiator
+
+on:
+  push:
+  pull_request:
+    branches: [main]
+  repository_dispatch:
+
+jobs:
+  check-plugin:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set environment variables
+        run: |
+          echo "GIT_SHA=${GITHUB_SHA}" >> $GITHUB_ENV
+          echo "GIT_BRANCH=${GITHUB_REF##*/}" >> $GITHUB_ENV
+      - name: Use environment variables
+        run: |
+          echo "GIT_SHA=${GIT_SHA}"
+          echo "GIT_BRANCH=${GIT_BRANCH}"
+      - name: Dispatch initiating repository event
+        if: github.event.action != 'response'
+        run: |
+          curl -X POST https://api.github.com/repos/RoadieHQ/plugins-verification/dispatches \
```

Comment:
This might be some use https://github.com/marketplace/actions/workflow-dispatch ? Might simplify sending inputs (i.e. PR branch) to plugins-verification (feel free to ignore this, just an observation(